src/Shared/FileUtilities.cs

```diff
@@ -4,6 +4,8 @@
 using System;
 #if !CLR2COMPATIBILITY
 using System.Collections.Concurrent;
+#else
+using Microsoft.Build.Shared.Concurrent;
```

Comment:
Can you please revert this unrelated change?

Reply:
Moved to a separate PR.